libs/api/ebpf_api.cpp

```diff
@@ -3785,28 +3786,42 @@ _Requires_lock_not_held_(_ebpf_state_mutex) _Ret_maybenull_
 }
 CATCH_NO_MEMORY_PTR(struct bpf_object*)
 
-_Ret_maybenull_ struct bpf_program*
-ebpf_program_next(_In_opt_ const struct bpf_program* previous, _In_ const struct bpf_object* object) NO_EXCEPT_TRY
+// This function is derived from libbpf's function of the same name.
+static _Ret_maybenull_ struct bpf_program*
+__bpf_program__iter(_In_opt_ const struct bpf_program* program, _In_ const struct bpf_object* object, bool forward)
```

Comment:
nit: should this function be in one of the libbpf_* files?

Reply:
It's static, I didn't see any value in exporting it just for use in 1 other file since it's also internal in libbpf.